joomla/joomla-cms
# [4.1] Workflow Stage display

## Purpose
If you are using workflowsand  viewing a list of articles it is not possible to easily see what stage each article is at. You can hover over the icon (which appears to be disabled but thats another issue) to find out the stage for each article but you cannot easi;y see an overview.  And if you sort by Stage then you have no idea how they are sorted etc etc

### Test Setup Instructions

- Enable workflows in the content options
- Create several articles and set them at different stages
- Also check the deatured articles view

### Before this PR
No visible means to see the stage of all the articles
![image](https://user-images.githubusercontent.com/1296369/166962775-f37f9370-daff-4b60-934d-1ef7f97c8e80.png)

### After this PR
The stage is now displayed
![image](https://user-images.githubusercontent.com/1296369/166962679-58f3bfba-a65f-4e32-a491-166a65aa44ba.png)
